### PR TITLE
fix 1414625. Additional fix to run password commands on control node

### PR DESCRIPTION
**Pull request by jcantrill**
fixes where we check for tool on control, but were still running on master

@ewolinetz @sdodson

### CODE REVIEW

**Comment by jcantrill** [comment]
aos-ci-test

File: roles/openshift_metrics/tasks/generate_hawkular_certificates.yaml [hunk 1]
```diff
@@ -19,25 +19,53 @@
...
+    dest: "{{openshift_metrics_certs_dir}}/{{item}}"
+  with_items:
+  - hawkular-metrics.pwd
+  - hawkular-metrics.htpasswd
+  - hawkular-jgroups-keystore.pwd
+  when: "not pwd_files['{{ item }}'].exists"
 
 - include: import_jks_certs.yaml
 
+- file: path="{{local_tmp.stdout}}" state=absent
```
Comment (ewolinetz) [inline roles/openshift_metrics/tasks/generate_hawkular_certificates.yaml]: this should probably be a local_action since we create it as a local_action

**Comment by ewolinetz** [comment]
one nit, otherwise LGTM

File: roles/openshift_metrics/tasks/generate_hawkular_certificates.yaml (hunk 1 above)
Comment (jcantrill) [inline roles/openshift_metrics/tasks/generate_hawkular_certificates.yaml]: fixed

**Comment by jcantrill** [comment]
@sdodson  yes

**Comment by jcantrill** [comment]
aos-ci-test

**Pull request merged**